drivers/atwinc15x0/Kconfig

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_ATWINC15X0
+    bool "ATWINC15x0 WiFi module"
+    depends on HAS_PERIPH_GPIO
+    depends on HAS_PERIPH_GPIO_IRQ
+    depends on HAS_PERIPH_SPI
+    depends on TEST_KCONFIG
+    select PACKAGE_DRIVER_ATWINC15X0
+    select MODULE_PERIPH_GPIO
+    select MODULE_PERIPH_GPIO_IRQ
+    select MODULE_PERIPH_SPI
+    select MODULE_NETDEV_ETH
```

Comment:
I'm sorry but I think this is not modelled yet. That's the main reason we did not move yet to modelling networking drivers.

Reply:
`drivers/net/netdev` seems not to be so much complicated. Since there were no error messages, maybe we should keep it so it won't be forgotten later?